code/modules/surgery/organs/internal/heart/_heart.dm

```diff
@@ -285,7 +285,7 @@
 //surplus organs are so awful that they explode when removed, unless failing
 /obj/item/organ/internal/heart/cybernetic/surplus/Initialize(mapload)
 	. = ..()
-	AddElement(/datum/element/dangerous_surgical_removal)
+	AddElement(/datum/element/dangerous_organ_removal, /*surgical = */ TRUE)
```

Comment:
why comment this out?

Reply:
Positional arguments don’t work un AddElement iirc and I still want it to be clear in a glance